### PR TITLE
:bookmark: (23.4.0) release notes

### PR DESCRIPTION
**Pull request by MatissJanis**
**Actual has now been moved to a stand-alone Docker organization. If you were previously using `jlongster/actual-server` docker image - please update it to `actualbudget/actual-server`.**

The release has the following notable features:

- Rules can now optionally be applied when any of their conditions match (in addition to the existing option to apply when all of their conditions match)
- Rules: quick-create option from the transaction table (in the “X selected” menu that shows up after selecting a transaction, choose “Create rule”)
- Ability to hide decimal places for currencies with large numbers (in Settings → Formatting)
- New autocomplete component (please report any bugs [here](https://github.com/actualbudget/actual/issues/773))
- Lots of smaller improvements and bugfixes

### CODE REVIEW

File: docs/Release-Notes.md [hunk 1]
```diff
@@ -2,9 +2,105 @@
...
+## 23.4.0
+
+**Docker tag: 23.4.0**
+
+The release has the following notable features:
+
+- Rules: ability to construct "any"/"all" conditions
+- Rules: quick-create option from the transaction table
+- Ability to hide decimal places for currencies with large numbers
+- New autocomplete component (report bugs [here](https://github.com/actualbudget/actual/issues/773))
```
Comment (j-f1) [inline docs/Release-Notes.md:14]:
```suggestion
- New autocomplete component (please report any bugs [here](https://github.com/actualbudget/actual/issues/773))
```

File: docs/Release-Notes.md [hunk 2]
```diff
@@ -2,9 +2,105 @@
...
 
+## 23.4.0
+
+**Docker tag: 23.4.0**
+
+The release has the following notable features:
+
+- Rules: ability to construct "any"/"all" conditions
+- Rules: quick-create option from the transaction table
+- Ability to hide decimal places for currencies with large numbers
```
Comment (j-f1) [inline docs/Release-Notes.md:13]:
```suggestion
- Ability to hide decimal places for currencies with large numbers (in Settings → Formatting)
```

File: docs/Release-Notes.md [hunk 3]
```diff
@@ -2,9 +2,105 @@
 title: Release Notes
 ---
 
+## 23.4.0
+
+**Docker tag: 23.4.0**
+
+The release has the following notable features:
+
+- Rules: ability to construct "any"/"all" conditions
```
Comment (j-f1) [inline docs/Release-Notes.md:11]:
```suggestion
- Rules can now optionally be applied when any of their conditions match (in addition to the existing option to apply when all of their conditions match)
```

File: docs/Release-Notes.md [hunk 4]
```diff
@@ -2,9 +2,105 @@
...
+- Rules: quick-create option from the transaction table
```
Comment (j-f1) [inline docs/Release-Notes.md:12]:
```suggestion
- Rules: quick-create option from the transaction table (in the “X selected” menu that shows up after selecting a transaction, choose “Create rule”)
```

File: docs/Release-Notes.md [hunk 5]
```diff
@@ -297,7 +393,7 @@ Version: 23.2.5
 - [#112](https://github.com/actualbudget/actual-server/pull/112) build(deps): bump moment from 2.29.3 to 2.29.4
 - [#117](https://github.com/actualbudget/actual-server/pull/117) build(deps): bump http-cache-semantics from 4.1.0 to 4.1.1
 - [#118](https://github.com/actualbudget/actual-server/pull/118) ⬆️ upgrade @actual-app/api to 4.1.6: node-fetch v2 support — thanks [MatissJanis]
-- [#119](https://github.com/actualbudget/actual-server/pull/119) ⬆️ upgrade express\*, bcrypt and body-parser — thanks [MatissJani]s
+- [#119](https://github.com/actualbudget/actual-server/pull/119) ⬆️ upgrade express\*, bcrypt and body-parser — thanks [MatissJanis]
```
Comment (j-f1) [inline docs/Release-Notes.md:396]: 😬 oops!

**Review by j-f1 — APPROVED** [review]

**Review by j-f1 — APPROVED** [review]

File: docs/Release-Notes.md [hunk 6]
```diff
@@ -98,6 +98,32 @@ Version: 23.4.0
...
+
+- [#167](https://github.com/actualbudget/actual-server/pull/167) Fix config.json in a default location getting silently ignored when it contains syntax errors. — thanks [Jackenmen]
+
+#### Maintenance
+
+- [#150](https://github.com/actualbudget/actual-server/pull/150) Update `docker-compose.yml` to clarify proper usage in production — thanks [j-f1]
+- [#165](https://github.com/actualbudget/actual-server/pull/165) Add support for automatically generating release notes — thanks [j-f1]
+- [#168](https://github.com/actualbudget/actual-server/pull/168) Expose demo bank that can be used to test Nordigen bank-sync — thanks [MatissJanis]
+- [#171](https://github.com/actualbudget/actual-server/pull/171) Fix app-sync.test.js not being ran due to faulty jest configuration — thanks [Jackenmen]
+- [#175](https://github.com/actualbudget/actual-server/pull/175) Push Docker images to new `actualbudget` Docker Hub organization. — thanks [trevdor]
```
Comment (j-f1) [inline docs/Release-Notes.md:125]: maybe call this out in the highlights?

Comment (MatissJanis) [reply docs/Release-Notes.md:131]: Makes sense. Added a big info banner

**Review by j-f1 — APPROVED** [review]

**Pull request merged**